src/ireiat/config.py

```diff
@@ -56,10 +56,21 @@
 }
 
 # TAP-related parameters
+# Highway
 HIGHWAY_CAPACITY_TONS = (
     80000  # TODO (NP): Placeholder while we translate from tons->vehicles or put a tonnage limit
 )
 HIGHWAY_BETA = 4  # the exponent in the congestion term
 HIGHWAY_ALPHA = 0.15  # the scalar in the congestion term
 
+# Rail
+RAIL_CAPACITY_TONS = 100000
+RAIL_BETA = 3.5  # the exponent in the congestion term
+RAIL_ALPHA = 0.1  # the scalar in the congestion term
 RAIL_DEFAULT_MPH_SPEED = 20  # miles per hour
+
+# Marine
+MARINE_CAPACITY_TONS = 150000
+MARINE_BETA = 3  # the exponent in the congestion term
+MARINE_ALPHA = 0.05  # the scalar in the congestion term
```

Comment:
Could argue that this would be near zero (which is fine), which makes this problem effectively "collapse" into a shortest path problem. Fine leaving it here but worth commenting.

I also think it's worth putting some of these parameters in the docs. Let's talk about it...